src/docker-compose

```diff
@@ -154,7 +154,7 @@ comp_start() {
         cd "$manifests_dir"
         $DOCKER_COMPOSE_CMD \
             --project-name "$project_name" \
-            up -d > ../compose.log 2>&1
+            up --detach > ../compose.log 2>&1
```

Comment:
If I remember correctly, this doesn't work with the `docker-compose` (Python-based) implementation. I'd hate to drop support for it just because of this, but YMMV.